packages/back-end/src/models/InformationSchemaTablesModel.ts

```diff
@@ -0,0 +1,66 @@
+import omit from "lodash/omit";
+import z from "zod";
+import mongoose from "mongoose";
+import { InformationSchemaTablesInterface } from "../types/Integration";
+import { errorStringFromZodResult } from "../util/validation";
+import { logger } from "../util/logger";
+
+const informationSchemaTablesSchema = new mongoose.Schema({
+  organization: {
+    type: String,
+    index: true,
+  },
+  table_name: String,
+  table_schema: String,
+  database_name: String,
```

Comment:
Our keys in mongo are usually camelCase.  Either way, we should be consistent within this single model (e.g. `table_name` and `dateCreated`)

Reply:
Updated to camelCase here and within the `InformationSchemaTablesModel` as well.